content/03.use-cases-a.md

```diff
@@ -1,9 +1,9 @@
-## GitHub in EcoEvo examples (Part 1)
```

Comment:
Great, yeah I think it'll be nice to keep some of the headings, but we can definitely remove all these "part 1"